public/javascripts/SVValidate/src/keyboard/Keyboard.js

```diff
@@ -82,11 +83,44 @@ function Keyboard(menuUI) {
         }
     }
 
+    function _setSeverity(severity) {
+        svv.ui.newValidateBeta.severityMenu.find(`#severity-button-${severity}`).click();
+    }
+
+    function _setDisagreeReason(reasonNumber) {
+        if(svv.newValidateBeta){   
+            svv.ui.newValidateBeta.disagreeReasonOptions.find(`#no-button-${reasonNumber}`).click();
+        } else {
+            menuUI.disagreeReasonOptions.find('.validation-reason-button').removeClass('chosen');
+        }
+    }
+
+    function _setUnsureReason(reasonNumber) {
+        if(svv.newValidateBeta){    
```

Comment:
Make sure to take a peek at the [Style Guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-guide)! The spacing we're looking for is more like
```
if (svv.newValidateBeta) {
```